viam-cartographer.go

```diff
@@ -750,6 +756,11 @@ func (cartoSvc *CartographerService) DoCommand(ctx context.Context, req map[stri
 		cartoSvc.logger.Warn("DoCommand called after closed")
 		return nil, ErrClosed
 	}
+
+	if _, ok := req["job_done"]; ok {
+		return map[string]interface{}{"job_done": cartoSvc.jobDone.Load()}, nil
```

Comment:
Have you manually tested through the `viam` cli that this works as expected both before & after the job is done?

Reply:
sorry, just saw this in the PR description. Answer was yes

Reply:
Yes, heres the command:

```
~ viam --base-url=https://pr-2017-appmain-bplesliplq-uc.a.run.app:443 robot part run --robot kims-robot --location "First Location" --organization "Nils's Org" -part kims-robot-main  -d '{"name": "slam-svc", "command": {"job_done": "true"}}' viam.service.slam.v1.SLAMService.DoCommand
```

Here's the output before the log:
```
{
  "result": {
      "job_done": false
    }
}
```

after the log:
```
{
  "result": {
      "job_done": true
    }
}
```

and after reconfiguring fails:
```
ERROR:
  Code: Unknown
  Message: resource (viam:slam:cartographer) is closed
```